src/common/vfs.h

```diff
@@ -44,6 +44,12 @@ struct OCSYNC_EXPORT VfsSetupParams
      */
     QString filesystemPath;
 
+    // Folder display name in Windows Explorer
+    QString displayName;
```

Comment:
I think you can get to that information (same thing for `alias`) from `FolderMan` by using the `filesystemPath`. It's better to get fresh information straight from the `Folder` object each time because they can change over time after that VfsSetupParams has been created.

Reply:
@er-vin I've been trying to avoid having a dependency to a **FolderMan**. We already have a dependency from GUI to VFS, so I thought two-way dependency is not a very good idea?

Reply:
@er-vin One other thing is - I actually don't need a refreshed information about the alias. I am just using an alias to be able to set multiple sync folders for the same user. So it is used as the last part of **syncRootId** so each folder can have the same user account and Windows Sid, but different alias at the end (!0, !1, etc.). From what I have noticed - the alias is never changed. Unless the folder is removed and then another folder is added. Then the alias is reused. But, I am removing the Registry key for that folder that's being removed anyway. When unregistering the Registry key - I STILL need the same alias that I've used when registering it. Otherwise, the Registry key I am trying to unregister is noexisting.